src/Lecture1.hs

```diff
@@ -29,98 +29,38 @@ module Lecture1
     , subString
     , strSum
     , lowerAndGreater
-    ) where
+	, check
```

Comment:
I see that a Tab character has been used here. Unfortunately, it's a warning in GHC ⚠️ 
```suggestion
    , check
```